doc/README.rst

```diff
@@ -5,9 +5,16 @@ This directory contains source and tools to build the GA4GH schemas
 documentation.
 
 The documentation is built automatically by Read the Docs upon commit
-to the master branch.  The resulting docs should appear at
-https://ga4gh-schemas.readthedocs.org/ within 2 minutes after commit.
+to the master and documentation branches.  See:
 
+The `resulting docs <http://ga4gh-schemas.readthedocs.org/>`_ are
+built automatically at Read the Docs, usually within two minutes of a
+commit.
+
+.. note:: While documentation is under development in the
```

Comment:
This note can go away, since there won't be a documentation branch after we merge.